templates/web/common/includes/list_of_products.tt.html

```diff
@@ -113,20 +113,9 @@
             [% product.product_name %]
           </a>
         </li>
-      [% ELSE %]
-        <li>
-        [% IF product.product_name == '' %]
-          <a href="[% product.url %]" title="[% product.code %]">
-          [% ELSE %]
-          <a href="[% product.url %]" title="[% product.product_name %]">
-          [% END %]
-            <div>[% product.img %]</div>
-            <span>[% product.product_name %]</span>
-          </a>
-        </li>
-        [% END %]
```

Comment:
Maybe add a message (to make it clear for people not having JS activated in some way):
```html
     [% ELSE %]
       <div>Product list loading from javascript…</div>
     [% END %]
```